src/main/java/seedu/duke/command/EditCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.command;
+
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
+
+import java.math.BigDecimal;
+
+/**
+ * The command that edits a selected item.
+ */
+public class EditCommand extends Command {
+    private final Item selectedItem;
+    private final Item updatedItem;
+    private static final String UPDATE_COMPLETE_MESSAGE = "This item has been updated."; //to be added to UI part later
+
+
+    /**
+     * The EditCommand constructor.
+     * @param name the name of the selected item
+     * @param purchaseCost the new cost of the item
+     * @param sellingPrice the new price of the item
+     * @param list the itemContainer where the selected item is stored in
+     */
+    public EditCommand(String name, String purchaseCost, String sellingPrice, ItemContainer list) {
+        this.selectedItem = list.getItem(name);
+        this.updatedItem = new Item(name, purchaseCost, sellingPrice);
+    }
```

Comment:
Item has set cost and price method, can just update directly, but this implementation also works.